packages/hooks/use-aria/index.ts

```diff
@@ -0,0 +1,19 @@
+import { buildProps } from '@element-plus/utils'
+
+export const useAriaProps = buildProps({
+  /**
+   * @description native `aria-label` attribute
+   */
+  ariaLabel: String,
+  /**
+   * @description native `aria-orientation` attribute
+   */
+  ariaOrientation: {
+    type: String,
+    values: ['horizontal', 'vertical', 'undefined'],
+  },
+  /**
+   * @description native `aria-controls` attribute
+   */
+  ariaControls: String,
```

Comment:
`ariaOrientation` and `ariaControls`prop appear to be useless in most components modified above. Perhaps it should be treated differently here.

Reply:
In my opinion, `aria-xxx` means native props with no ambiguity.. If you try to differentiate by passing a parameter instead, it becomes cumbersome.

Reply:
For example, the current `el-rate` code shows that the `aria-controls` and `aria-orientation` settings are supported, but even if they are set, there seems to be no logic to add related attributes, which seems wrong. May cause confusion to users.

Reply:
Just don't write it in the document.

Reply:
Yes, I noticed. However, ts may prompt you in the code editor. I think it is better to remove it directly.

Reply:
Maybe we can add a logic to pick props